pkg/graph/schema.resolvers.go

```diff
@@ -383,6 +383,18 @@ func (r *mutationResolver) UpdateSystemIntakeAdminLead(ctx context.Context, inpu
 	}, err
 }
 
+func (r *mutationResolver) UpdateSystemIntakeReviewDates(ctx context.Context, input model.UpdateSystemIntakeReviewDatesInput) (*model.UpdateSystemIntakeReviewDatesPayload, error) {
+	_, err := r.store.UpdateReviewDates(ctx, input.ID, input.GrbDate, input.GrtDate)
+	systemIntake := models.SystemIntake{
+		GRBDate: input.GrbDate,
+		GRTDate: input.GrtDate,
+		ID:      input.ID,
+	}
+	return &model.UpdateSystemIntakeReviewDatesPayload{
+		SystemIntake: &systemIntake,
```

Comment:
It looks like this is only returning a few fields on the system intake. Shouldn't it return the entire intake? Don't we get the intake back from UpdateReviewDates?